src/html.tsx

```diff
@@ -65,6 +65,8 @@ export default function HTML(props: HTMLProps): JSX.Element {
                                 person_profiles: 'identified_only',
                                 __preview_heatmaps: true,
                                 opt_in_site_apps: true,
+                                __preview_remote_config: true,
```

Comment:
new flags, as of now, requires using the remote config (which is stable and used on us.posthog.com)